Makefile

```diff
@@ -198,7 +198,7 @@ lint-flags:
 
 .PHONY: generate
 generate: ## Re-generate generated code and documentation
-generate: generate-rbac generate-crd-deepcopy generate-crd-yaml generate-deployment generate-api-docs generate-metrics-docs generate-uml
```

Comment:
After this change, how do we update the metrics docs? They're still present on the "Prometheus metrics" page, but it seems like now we won't be able to manage them programmatically? If the plan is to fix them up later, that's fine, but a TODO here or an issue link would be great.

Reply:
Why do we need to dynamically generate that page? It rarely changes and is just overhead to me. Additionally, the bits to generate the docs needed to be added by hand.

Reply:
removing it also means we don't run promlint on our metrics, which could mean we add something not ideal as well

Reply:
https://github.com/projectcontour/contour/issues/3717